test/integration/test_http_client.sh

```diff
@@ -1,5 +1,21 @@
 #!/bin/bash
 set -e
+
+WAITTIME=1
```

Comment:
I think it is better to just copy this script somewhere if you want to use it for opengauss testing. There is no use for a wait time for normal platform integration tests and for opengauss testing you don't want the tests to ever run. You can use "sleep infinity" then the components will run until you use ctrl-c to clean it up

Reply:
Added a new script to send commands remotely from opengauss.